packages/docusaurus-mdx-loader/src/index.js

```diff
@@ -25,9 +25,20 @@ module.exports = async function(fileString) {
   const callback = this.async();
 
   const {data, content} = matter(fileString);
-  const options = Object.assign(DEFAULT_OPTIONS, getOptions(this), {
+  const reqOptions = getOptions(this) || {};
+  const options = {
+    ...reqOptions,
+    remarkPlugins: [
+      ...DEFAULT_OPTIONS.remarkPlugins,
+      ...(reqOptions.remarkPlugins || []),
```

Comment:
Not sure if we should just replace the user's or add it on. How will users turn off plugins in this approach?

Reply:
im having a hard time on this as well. if they only add one plugin, they might remove the whole default plugin.

not sure whether to add or to completely reassign.

Reply:
I think let's stick with this. As long as we're sure that the default plugins we've chosen are useful and nobody will want to remove it.